packages/babel-generator/src/index.js

```diff
@@ -47,6 +47,8 @@ function normalizeOptions(code, opts): Format {
     minified: opts.minified,
     concise: opts.concise,
     jsonCompatibleStrings: opts.jsonCompatibleStrings,
+    numbers: opts.numbers,
+    lowercaseHex: opts.lowercaseHex,
```

Comment:
Sicne we plan to remove `jsonCompatibleStrings` because it can already be set using `jsescOption`, I don't think that we should introduce `numbers` and `lowercaseHex`.
Users can directly set them on `jsescOption`.

Reply:
Yeah, I talked to @sidntrivedi012 offline on this problem, he is working on a fix.